models/option.go

```diff
@@ -5,10 +5,42 @@ import (
 	"fmt"
 )
 
+// Type is to select the user interaction type that is required to fill an option
+type Type string
+
+// OptionTypes we currently support, list, user-input, optional-user-input
+const (
+	// Originally:
+	// - if there was only one key then this question was not asked
+	// - if there was more than one keys then it was asked in a list to select from
+	// Now, if this type is selected:
+	// - if there is only one key then this question shall not not be asked
+	// - if there are more than one keys then the selection must be asked in a list to select from
+	TypeSelector Type = "selector"
+	// Originally:
```

Comment:
If possible, update the comments to focus on the current expected behaviour (as historical one can be found in the git history)